index.html

```diff
@@ -3,14 +3,165 @@
+        <div class="mouseIcon bgicon"><i class="fas fa-mouse"></i></div>
+        <div class="skullIcon bgicon"><i class="fas fa-skull"></i></div>
+        <div class="skullIcon2 bgicon"><i class="fab fa-playstation"></i></div>
+        <div class="circleIcon "><img src="img/circleIcon.PNG" alt=""></div>
+      </div>
+
+      <div class="head_content">
+        <div class="headText">
+          <h3>"Hello! shering world"</h3>
+          <h1>Trampoline Dev Global Submit <br/> 2022</h1>
+       </div>
+       <div class="head_sup_text"> Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi odit
+          architecto, quis consectetur laborum dolorem voluptatem officiis! </div>
+       <h2>2022.02.10(THU) ~ 11(FRI)</h2>
+       <h4>@Global assembly, Game development and more</h4>  
+      </div>   
+    </section>
+
+    <!--Program Section-->
+
+    <section id="program">
+      <h3>Main Program</h3>
+      <div class="red_vector"></div>
+      <div class="program">
+      <div class="lecture program_item">
+        <div class="program_icon"><img src="img/lecture.png" alt="lecture icon"></div>
+        <div class="program_title">Lecture</div>
+        <div class="program_text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Hic excepturi, possimus expedita cum nisi
+        </div>
+      </div>
+      <div class="exhibition program_item">
+        <div class="program_icon"><img src="img/exhibition.png" alt="echibition icon"></div>
+        <div class="program_title">DT Exhibition</div>
+        <div class="program_text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Hic excepturi, possimus expedita cum nisi
+        </div>
+      </div>
+      <div class="forum program_item">
+        <div class="program_icon"><img src="img/forum.png" alt="forum icon"></div>
+        <div class="program_title">Forum</div>
+        <div class="program_text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Hic excepturi, possimus expedita cum nisi
+        </div>
+      </div>
+      <div class="workshop program_item">
+        <div class="program_icon"><img src="img/workshop.png" alt="workshop icon"></div>
+        <div class="program_title">Workshop</div>
+        <div class="program_text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Hic excepturi, possimus expedita cum nis
+        </div>
+      </div>
+      <div class="ignite program_item">
+        <div class="program_icon"><img src="img/ignite.png" alt="ignite icon"></div>
+        <div class="program_title">Ignite</div>
+        <div class="program_text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Hic excepturi, possimus expedita cum nisi
+        </div>
+      </div>
+    </div>
+
+      <button id="Program-btn">Join TD Global Summit 2022</button>
+      <div class="seeprogram"><a href="#">Se the whole program</a></div>
+    </section>
+    <!--feature spekear-->
+    <section id="features">
+    </section>
```

Comment:
Please ensure that the pages look almost identical to the original design.

Your Implementation | Figma Design
:---:|:---:
   
![your_design](https://user-images.githubusercontent.com/10332499/153350762-636e2713-e5d7-4014-be1e-05a43c5ec1ca.PNG) | ![design](https://user-images.githubusercontent.com/10332499/153350789-b65fb35c-78a8-445d-b02e-305b066e722d.PNG)